services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/manager/EntityAuthorizationManagerAutowireTest.java

```diff
@@ -754,16 +645,49 @@ public void testCanCreate() {
 		EntityType createType = EntityType.project;
 		UserInfo user = userTwo;
 		
-		// old call under test
-		AuthorizationStatus oldStatus = entityPermissionManager.canCreate(parentId, createType, user);
```

Comment:
We can even get rid of the dependency of entityPermissionManager in this test class